### PR TITLE
Sign transaction

### PR DESCRIPTION
**Pull request by theblockstalk**

### CODE REVIEW

File: src/demo/components/TextboxLayout.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const TextboxLayout = (props: TextboxLayoutProps) => {
         }}
       />
       <label htmlFor="inputField" className="textbox-label">
-        {props.label}
+        tes
```
Comment (sadiabbasi) [inline src/demo/components/TextboxLayout.tsx:22]: keep the {props.label} here

File: src/demo/components/styles.tsx [hunk 2]
```diff
@@ -25,16 +24,287 @@ export const PageIntroStyle = styled.div`
...
   border-radius: 10px;
   margin: 2rem 4rem 0rem;
-  border: 1px solid #E4E4E4;
+  border: 1px solid #e4e4e4;
   p {
     text-align: justify;
   }
+`;
+
+export const HeaderContainer = styled.div`
```
Comment (sadiabbasi) [inline src/demo/components/styles.tsx:40]: styles.tsx designed for the common style which is used in many pages otherwise create component based css

File: src/demo/module-index.tsx [hunk 3]
```diff
@@ -3,14 +3,16 @@ import ReactDOM from "react-dom/client";
...
 
 export default function initiate(root: ReactDOM.Root) {
   return root.render(
     <React.StrictMode>
       <ErrorHandlerProvider />
 
       <ProSidebarProvider>
-        <Router />
+        {/* <Router /> */}
+        <BlockchainTx />
```
Comment (sadiabbasi) [inline src/demo/module-index.tsx:15]: revert this change

File: src/demo/pages/BlockchainTx.tsx [hunk 4]
```diff
@@ -111,133 +122,100 @@ export default function BlockchainTx() {
...
-                </div>
-              </div>
+    <>
+      <HeaderContainer>
+        <HeaderPictureContainer></HeaderPictureContainer>
+        <FeatureNameLabel>Feature Name: Sign Transaction</FeatureNameLabel>
+
+        <HowToUseLabel>How to use :</HowToUseLabel>
+        <HeaderTonomy>Tonomy ID</HeaderTonomy>
+        {/* <HeaderTonomyID>Tonomy ID</HeaderTonomyID> */}
```
Comment (sadiabbasi) [inline src/demo/pages/BlockchainTx.tsx:132]: remove comments

File: src/demo/pages/BlockchainTx.tsx [hunk 5]
```diff
@@ -111,133 +122,100 @@ export default function BlockchainTx() {
...
+        </HeaderDescription>
+
+        <Link href="/Test">learn about the Antelope blockchain protocol </Link>
+        <br />
+        <br />
+        <Button variant="outlined" size="large">
+          Enter Demo
+        </Button>
+      </HeaderContainer>
+      <MainDescription>
```
Comment (sadiabbasi) [inline src/demo/pages/BlockchainTx.tsx:146]: it's better to use the paragragh and heading tags instead of such tags and use component level css

File: src/demo/routes/mainRoutes.tsx [hunk 6]
```diff
@@ -5,26 +5,26 @@ import UserHome from "../pages/UserHome";
 import W3CVCs from "../pages/W3CVCs";
```
Comment (sadiabbasi) [inline src/demo/routes/mainRoutes.tsx:5]: revert this file changes

**Review by sadiabbasi — CHANGES_REQUESTED** [review]

**Review by YAS-SIIN — COMMENTED** [review]
Reviewed

File: src/demo/components/TextboxLayout.tsx (hunk 1 above)
Comment (YAS-SIIN) [reply src/demo/components/TextboxLayout.tsx:22]: I removed this component. I no longer need this.

File: src/demo/components/styles.tsx (hunk 2 above)
Comment (YAS-SIIN) [reply src/demo/components/styles.tsx:40]: I created some common styles which we will use in the other pages. If you want, I can separate it.

File: src/demo/routes/mainRoutes.tsx (hunk 6 above)
Comment (YAS-SIIN) [reply src/demo/routes/mainRoutes.tsx]: Done

File: src/demo/pages/BlockchainTx.tsx (hunk 5 above)
Comment (YAS-SIIN) [reply src/demo/pages/BlockchainTx.tsx:182]: Done

File: src/demo/pages/BlockchainTx.tsx (hunk 4 above)
Comment (YAS-SIIN) [reply src/demo/pages/BlockchainTx.tsx]: Done

File: src/demo/components/styles.tsx (hunk 2 above)
Comment (YAS-SIIN) [reply src/demo/components/styles.tsx:40]: I changed some of the styles.

File: src/demo/components/TextboxLayout.tsx (hunk 1 above)
Comment (YAS-SIIN) [reply src/demo/components/TextboxLayout.tsx]: Done

File: src/demo/pages/BlockchainTx.tsx [hunk 7]
```diff
@@ -25,32 +20,75 @@ import {
...
+    label: "Fetching sovereign signer and checking if the key is still valid",
+  },
+  {
+    label: "Signing transaction",
+  },
+  {
+    label: "Broadcasting to the Blockchain network",
+  },
+  {
+    label: "Confirming network",
```
Comment (theblockstalk) [inline src/demo/pages/BlockchainTx.tsx:55]: "Confirmed by receiving node"

File: src/demo/pages/BlockchainTx.tsx [hunk 8]
```diff
@@ -25,32 +20,75 @@ import {
...
+  },
+  {
+    label: "Transacting consensus on the network",
```
Comment (theblockstalk) [inline src/demo/pages/BlockchainTx.tsx:58]: "Transaction consensus on all nodes in the network"

File: src/demo/pages/BlockchainTx.tsx [hunk 9]
```diff
@@ -103,6 +141,9 @@ export default function BlockchainTx() {
         ? settings.config.blockchainUrl
         : "http://localhost:8888";
 
+      setActiveStep(4);
+      setProgressValue(100);
```
Comment (theblockstalk) [inline src/demo/pages/BlockchainTx.tsx:145]: change to
```ts
setActiveStep(3);
setProgressValue(80);

await sleep (3000) // 3 seconds

setActiveStep(4);
setProgressValue(100);
```

**Review by sadiabbasi — APPROVED** [review]
